api/BluetoothRemoteGATTService.json

```diff
@@ -32,7 +32,7 @@
           "edge": [
             {
               "version_added": "79",
-              "notes": "macOS only."
+              "notes": "macOS and Windows 10."
```

Comment:
Two things here:

- Could we make this change consistent with the other Chromiums?
- Can we revise these notes to be more explicit? Something like "Supported on Windows 10 and later and macOS." or "Supported by default only on Windows 10 and macOS." (whatever the case is).